### DISCUSSION TITLE
How can I make anchor headings generate for only h1-h6

### DISCUSSION THREAD

**Discussion by ghost** [allejo/jekyll-anchor-headings#40] [Q&A] [answered]
I want to only make anchor headings generate for h2-h6. Is there any way to do this?

**Comment by allejo** [comment, marked as answer]
Yes! There is the `h_min` and `h_max` arguments you can use. Set the minimum to 2 and the max to 6 or omit and it'll default to 6.


e.g.,

```liquid
{% include anchor_headings.html h_min=2 %}
```

**Reply by ghost** [reply]
Thank you!
